app/src/main/java/com/edu/uz/currency/currencyapp/retrofit/pojo/SingleCurrency.java

```diff
@@ -0,0 +1,54 @@
+
+package com.edu.uz.currency.currencyapp.retrofit.pojo;
+
+import java.util.List;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class SingleCurrency {
```

Comment:
Zazwyczaj co widziałem to były te klasy obiektów w postaci takiej jak przychodzi JSON czyli to co tu, w paczce "rest" i też tam mapper na zwykłe obiekty, no i klasy zwykłych obiektów które używamy później / zapisujemy w bazie.